docs/admin/monitoring/grafana.md

```diff
@@ -0,0 +1,57 @@
+# Services Dashboards 
+
+## Introduction
+
+The Hopsworks platform is composed of different services. Hopsworks uses Prometheus to collect health and performance metrics from the different services and Grafana to display them to the Hopsworks administrators. 
+
+In this guide you will learn how to access the Grafana dashboards to monitor the health of the cluster or to troubleshoot performance issues.
+
+## Prerequisites
+
+To access the services dashboards in Grafana, you need to have an administrator account on the Hopsworks cluster.
+
+## Step 1: Access Grafana 
+
+You can access the admin page of your Hopsworks cluster by clicking on your name, in the top right corner, and choosing `_Cluster Settings_ from the dropdown menu.
+
+You can then navigate to the _Monitoring_ tab. The _Monitoring_ tab gives you access to several of the observability tools that are already deployed to help you manage the health of the cluster.
+
+<figure>
+  <a  href="../../../assets/images/admin/monitoring/monitoring_tab.png">
+    <img src="../../../assets/images/admin/monitoring/monitoring_tab.png" alt="monitoring tab" />
+  </a>
+  <figcaption>Monitoring tab</figcaption>
+</figure>
+
+Click on the _Grafana_ link to open the Grafana web application and navigate through the dashboards.
+
+## Step 2: Navigate through the dashboards
+
+In the Grafana web application, you can click on the _Home_ button on the top left corner and navigate through the available dashboards.
+
+Dashboards are organized into three folders:
+
+- **Hops**: This folder contains all the dashboards of the Hopsworks services (e.g. the web application, the file system, resource manager) as well as the dashboards of the hosts (e.g. EC2 instances, virtual machines, servers) on which the cluster is deployed.
```

Comment:
If it's a simple enough thing to do it would be nice to rename this group of dashboards from Hops to Hopsworks.

Reply:
And the Hopsworks dashboard to Glassfish.

Reply:
But no need to block this PR.